### ISSUE TITLE
Debug problem, what a sad day, I don't know waht to do. 

### ISSUE THREAD

**Issue by andrea271068** [clough42/electronic-leadscrew#132]
Hi, finally today I complete my set-up, I was so happy, the time to try everything was ready.
I run code composer studio 9 on Mec Book.
I click the hammer, the debug, after that, I click the bug icon, but there the problem comes, they force me to download an update.
The control panel never show-up, when I try to do again the debug, another message shows up. 
I'm very lost, I don't understand please somebody can help me, I'm not a programmer, I use a computer every day but as a normal user, for the business not as a programmer.
Really I don't know how to fix this issue. 

Please check the attached photo, may somebody can help me to solve this thing. 

![1](https://user-images.githubusercontent.com/75749307/107888682-2ec7bf80-6f0e-11eb-8aee-02b57aaaaee6.jpg)
![2](https://user-images.githubusercontent.com/75749307/107888683-2ff8ec80-6f0e-11eb-9720-09ef1dc44d10.jpg)
![3](https://user-images.githubusercontent.com/75749307/107888684-2ff8ec80-6f0e-11eb-829c-82aa0165b9b3.jpg)
![4](https://user-images.githubusercontent.com/75749307/107888685-30918300-6f0e-11eb-81be-a3ed121f86ab.jpg)
![5](https://user-images.githubusercontent.com/75749307/107888686-30918300-6f0e-11eb-971b-93d476c6e5a0.jpg)
![6](https://user-images.githubusercontent.com/75749307/107888687-312a1980-6f0e-11eb-8c6d-6422ee7d97f4.jpg)

**Comment by andrea271068** [comment]
I forgot to add, I click resume, but when I click nothing happens, and if I try to do resume gain a message shows up and says that I have to complete the debugging process, really don't know??
Can be a wrong connection? can be that I have to do something on the program?

**Comment by poulbo** [comment]
You have to update XDS110 firmware with xdsdfu.exe at the suggestions  location (search for the location) 
also read the TI. 's pdf file

**Comment by andrea271068** [comment]
Many Thanks,
I can’t find xdsdfu.exe, or I find many things regarding XDS110 firmware on TI, please can you send to me the right link?, to many information and options find.

Waiting for your kind feedback, thanks is advance.

Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by poulbo** [comment]
Hi Andrea

 

I have the same problem with mine, and  I just do what they tell to do,

Here is the link  C:\ti\ccs930\ccs\ccs_base\common\uscif\xds110

I am using   firmware_3.0.0.14.bin     

The  XDS110SupportReadMe.pdf  tell you how to do it.

 

Hope it’s works

 

poulbo

**Comment by andrea271068** [comment]
Thanks, but the link doesn’t work >
C:\ti\ccs930\ccs\ccs_base\common\uscif\xds110

May there is an error and the typing

But I have to buy this hardware and install it?, if yes, how you did it? Please.


[cid:image002.jpg@01D70445.AF57CB60]

Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by poulbo** [comment]
Hi 

 

Andrea

Nothing  wrong with link , if you don’t have the xds110 directory ,something wrong with your installation .

Xds110 is the debugger coming with the Code Composer studio 9.3.0.

(I am using windows 10)

 

poulbo

**Comment by andrea271068** [comment]
Soo, I don’t have to buy that device, nice? Because is already inside the LounchXL and CCS 9.3.0.

I should find the C:\ti\ccs930\ccs\ccs_base\common\uscif\xds110
Inside Code Composer studio 9.1.0 installed?

I run this software in Mac Book Apple, I think the directory is not C, may this is why I can’t find it, now I will find the directory letter for Apple.

I don’t have experience in programming, this is the problem, really, I walk in the dark, any help to see the light will be very welcome, other wise I think I spend a huge money for nothing, and I will be forcing to dismantle everything and put on the side, really I don’t know what to do.

Sorry for my way out, but I need a person that drive me step by step on this thing.


Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by andrea271068** [comment]
I findXDF110 inside the finder in mac book, and I found the folder XDF110 and also the readme file mentioned.



Now please what I must do?

[cid:image002.jpg@01D7044F.209370A0]

[cid:image006.jpg@01D7044F.209370A0]



Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by andrea271068** [comment]
Dear Poulbo, this is the screen shoot, I don’t know the controller don’t go, but after I resume I don’t receive any error message.
I change cables from the controller to the logic, I disconnect all others connection from the servo to the driver and I disconnect also the encoder. Just to make sure that didn’t was a cable problem.
James in his video install the firmware in front his computer, and just the controller was connected to the logic (TI-XL +Interface board)
I download from TI, the XDS emulation software (Emu pack) for mac. And I try to download inside the 3.1.0

But this is the result. Check below screen shoot.


[cid:image001.jpg@01D7045C.7D010480]
Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>


From: Andrea Pisani
Sent: Tuesday, February 16, 2021 10:33 AM
To: clough42/electronic-leadscrew <reply@reply.github.com>; clough42/electronic-leadscrew <electronic-leadscrew@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [clough42/electronic-leadscrew] Debug problem, what a sad day, I don't know waht to do. (#132)

I findXDF110 inside the finder in mac book, and I found the folder XDF110 and also the readme file mentioned.



Now please what I must do?

[cid:image005.jpg@01D7045C.7D010480]

[cid:image007.jpg@01D7045C.7D010480]



Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by poulbo** [comment]
Hi Andrea

 

I can’t see any pictures ,the first mail you send was ok and with pictures off you  system

Something wrong now, mabe you have to talk with a Mac user,

 

poulbo

**Comment by andrea271068** [comment]
This is the image, do the same also with window

Try to see now, please don’t leave me alone.

Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by poulbo** [comment]
Hi Andrea

 

The first mail in this tread was ok with all your picture , but after that you are sending lot of stuff on Italia in your mail 

I don’t what to do with it, and there is no pictures, sorry.

 

Every time I answer you I hat cut all that stuff out from the mail.

 

poulbo

**Comment by andrea271068** [comment]
Let’s try now.
I attached the photo here
May did you have a personal e-mail contact?

I found on the program the > C:\ti\ccs930\ccs\ccs_base\common\uscif\xds110
But now I have to invent how to manage 😊 I don’t know nothing about programming, I check also the PDF is like Chinese language for me, 0 😊


Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by poulbo** [comment]
Hi Andrea

 

 

In the PDF file it  says:       

Mac and Linux systems require libusb to be installed. Go to libusb.info for more details.

(you have to read information in the file)

 

My problem is that I am not using the Mac system, and don’t  how I works, that’s  why I think you have to find a 

Mac user in this forum for more help, or get help from a Mac user in your country ,an if you also have to change parameters in the code,

Maybe a person who familiar about programming in cpp.

 

poulbo

**Comment by andrea271068** [comment]
Hi, Thanks, 
I update the firmware, but still stocked, the control panel didn't work. 

this is the page where I'm stuck after clicking the resume (blue arrow), nothing happens to the controller. 

I try using CCS in windows and in Mac computer, both are updated, but is the same. 

As you can see, from the attached screen shoot photo,  I don't have any error message, I just stuck.  

I think That I have to do something, but...

**I need Mr. James Help,** 
I can go forward, everything is installed but...


![Screen Shoot debug](https://user-images.githubusercontent.com/75749307/108160197-ec0e0f00-70e8-11eb-97cc-e3ee989ae206.jpg)

**Comment by poulbo** [comment]
Hi Andrea

 

Well it seems that debugger part works now, do you have access to an oscilloscope, 

I want you to measure on pin STB, CLK, DIO on James PCB card, 

to see signals on all 3 pins, and if not move cable and measure again, 

if there is a signal now the control card is defective (luckily they are cheap) 

I have 2 control boards that are defective. 

 

poulbo

**Comment by poulbo** [comment]
Hi Andrea 

 

I forgot to say I got your picture this time.

 

poulbo

**Comment by andrea271068** [comment]
Thanks :) Pouldo, I don't have Oshilloscope but I have the multimeter tester, I can check the continuity, sure.

Wow two control panel defective..., the major problem is that takes a lot of time to arrive, I don't know why. 

Sure today after business when I back home, I will check.
I will let you know ASAP, thanks a lot. In a way, I prefer to know that the control panel is defective than having a logic problem that I can't solve because I'm completely out of programming competence.  
Thanks :) let's see

**Comment by andrea271068** [comment]
Hi Pouldo, I check the continuity in the PCB card and also the connection within the PCB card and connection cable to the Led&Key 5 pin connector to the board, and is working too. 
The only strange thing is when I control the continuity just in the Led&key from 5 input pins and the related connection in the board if I test STB, CLK, DIO doesn't, blink, no sound, but maybe is normal.
I decided to buy again the PCB and Led&Key board, never mind. 
Pouldo is normal that the led light in the Led&key board is on, even if is just connected to the PCB and of course the 5V, sometimes if I move the cable this led light switch to off, the 5V connection from the transformer pin and the PCB is not so fixed, but the receiver is well soldered, very strange, for this reasons I decide to buy again this two components. Anyway when I try to debug that light in the Led&key was on, and also about that, I'm not so sure, is normal that the led light is switch on from the start?  
Please tell me what you think to, fill free to give me your opinion by your experience, 1000 thanks

**Comment by poulbo** [comment]
Hi Andrea

 

It is normal for LED lights on the control board to always light up.

if you take 5 pin plugs off and measure on CLK, STB, DIO on James

board with your multimeter in 10 or 20V mode measure first between 

GND and 5V+ then of course it shows 5V, measure then between GND and DIO,

and it should show approx. 3.5V-4.8 V does the same with STB and CLK,

should also show approx. 3.5V-4.8 V. If this is ok try again with the plug inserted.

You should also now see the same measurements. If some of the 3 pins are close

to 0V- 0.5V there is something wrong with the control board. 

One of my control boards pulled the DIO connection to 0V. input on TM1638 chip

was short-circuited. 

 

You can buy control board in Germany and you will get it faster.

 

Yours sincerely

poulbo

**Comment by cad-quark** [comment]
Andrea271068.   I do not know enough about this design to be dangerous but . . .    
 I got the same message on a Windows10 desktop, and on a Windows8.1 desktop.  It also told me “Target not found” which persisted after several reloads of CCS, and ELS 1.301.   In addition, the USB device was not being recognized by either machine.  In Windows, the system reports a USB device with a sound on insertion, and a different sound as its removed.  Neither machine made the USB noise.  
In frustration, I removed my 5 volt power supply, and put back all three isolation jumpers next to the USB card connector.  The Win8.1 machine then made both insert and remove noises one after another.  I pulled the 5 volt jumper and the ‘connected’ sound occurred once only.  The card was then powered by the computers USB outlet.  I was then able to flash Ver 1.301.   
I think its was a voltage problem.   The USB computer jack is not able to power the LaunchXL card and the 5v LEDs  but can power the card alone enough to flash 1.301 to memory.   My power supply measured 5.4 volts, which must have been too much.  I later used an adjustable power supply set voltage to exactly to 5v after I removed the jumpers and again the computer and CCS recognized the card.  
Also I suggest you use the USB cable supplied with the LaunchXL card since its nice and short and has a lower voltage drop.

I used CCS 9.1 build 10, (CCS 10 choked) ELS Ver 1.301 and have them on a USB drive since I reloaded them so often.  I checked all the connections with the drawings provided.  I tried reading the CCS  and documentation and watch the TI videos.  Not much help, especially the one in Chinese (actually all of it is a language I do not know) .  I am no embedded controller engineer or C programmer but I have seen a lot of useless messages kicked out by programs when the programming cannot figure out what failed.  There is no way CCS or XDS110 can measure operating voltage, so it obviously must be the debug that failed.  That was a fib in my experience.

 I have just added a BEI encoder and can display RPM 
![F0BFC80D-01E6-4ADF-86A0-5961C5481A19](https://user-images.githubusercontent.com/75825422/108461721-dad81600-7240-11eb-93c7-ef88ff9ee6c2.jpeg)
when spun by a drill.

**Comment by cad-quark** [comment]
And yes, the one power light is on with 5v from any source.  The important point is you do not need 5 volts to build, compile, link or flash the Ti Launchxl-F280049c, C2000, Piccolo, Icepick, XDS110, TMS320F280049c microcontroller.  3.3 volts will do just fine.  What you could use is fewer f——- names for it.  The 5 volts is only required to use the hardware of James’s brilliant design and careful programming.  Spend some time looking at Youtube Electronic Lead screw systems and it soon is obivious how amazing the Clough42 system is.  All that function from a single card and the work to do 20 videos.  
      Bravo JC.

**Comment by andrea271068** [comment]
Thanks to everybody, today I was in the office, I will try Pouldo, thanks also to Bravo JC, I will read it, is complex and I try to configure-out , if I understood you say that 5v transformer is not stable enough for the pcb and can create conflict, today I bought Dc-Dc step out module,my is the cheep transformer that gave me also problem, I will connect now to the servo 36V  power supply and I will transfor to 5v true the DC-Dc step out module that stabilize also the power, let's try. Hope to solve, thanks guys please continue to support me 🙂




my 36V piwersupply

**Comment by cad-quark** [comment]
You can eliminate all booster connections ( green module you built ) and run the LaunchXL with the USB cable,  only leave the 3.3 volt and ground jumpers.  Remove the 5v middle jumper.  Your computer and CCS should detect a USB device to  flash ELS 1.301 on to.  The Windows machines make the insert USB noise and add an XDS110 com port to the Device Manager.  I do not know what MAC does.  ( maybe you can help me load CCS to the MAC M1 AIR ).
  Don't forget to hit the resume arrow after the flash.  If that works, you have eliminated any possible problems with your soldering or crimping.  When you add your booster board and hookups to the Launch XL, on adding 5V power, the blue LED 3 should glow and the program is running.  I do not remember if this LED is active before the flash, but when the program is screwed up, and hangs at ‘message1’, it blinks erratically like a tiny blue screen of death.  I had to hit the white reset button on the LaunchXL.   What do you expect when the icon for “Build” is a hammer and flash is a bug.  We need a pickaxe for debug.  ⛏
On flash, LED2 will light up and blink
 
![A60DEAC3-A48F-48D4-9F07-6731A23707A6](https://user-images.githubusercontent.com/75825422/108546872-596b9c80-72af-11eb-88c2-77aa71b65bd7.
 Any one know CCS well enough to trace program execution or insert break points on the PC?

**Comment by cad-quark** [comment]
![7D21A2AC-7258-43DD-A796-DB24B0537EA9](https://user-images.githubusercontent.com/75825422/108547801-93896e00-72b0-11eb-88ee-57ef00186a5a.jpeg)

**Comment by cad-quark** [comment]
What I am saying is the 5V you supply must be close to 5 v that is all.  You do not need anything but the base LaunchXL card and a USB cable to program.  You can add the Clough42 parts later.   I just now tweaked the program and changed my encoder to 4000 pulses rather than 4096.  I used only the Usb cable and was powered by the computeer's cable only.  The Clough42 parts were removed.  This is how most users of controller cards program anyway.Doing so eliminates any bad construction or missed solder joints.  -----Original Message-----

**Comment by andrea271068** [comment]
Get it! You say debug only connection the TI via USB to the computer, because I don't need the rest. After that I can still plug in the Clught42 control board?

**Comment by andrea271068** [comment]
Okay, I will try, I will debug, plug-in the TI directly via usb to the computer without PCB, I will debug and only after I will plug the Pcb board. 
Thanks, I will let you know 🙂

**Comment by cad-quark** [comment]
Right.  You win a bucket of Iowa Snow.  Let us know if it works for you

**Comment by cad-quark** [comment]
Oh yes, Bravo JC is just hooray for James Clough and his design.

**Comment by andrea271068** [comment]
Good morning, I did it, I bluild up flashing the code true computer Usb whitout the  PCB, saying buildup completed and running. After that I disconnect usb and I remove the three jumpers, and I connect again the PCB, but notting happen, I don't know, see the photo. 
How I can delete now the flash in the memory? And try again to build up? When I try to do it, gave me some errors. ![image](https://user-images.githubusercontent.com/75749307/108587564-989ff900-7354-11eb-8e84-22dde1304062.jpeg)

**Comment by andrea271068** [comment]
Hi, How I can erase previous flashed code on in the memory? i wan to debug again from 0

**Comment by cad-quark** [comment]
Make your changes and hit debug.  I only had "els-f280049c" under the debug.  I changed steps and encoder resolution and flashed the card again.  It erases the previous flash.  Do not forget to hit resume.   After that, you might want to exit CCS or you will get an error when you pull the USB cable...  -----Original Message-----

**Comment by cad-quark** [comment]
Were you able to program the LaunchXL processor?

**Comment by cad-quark** [comment]
Did you hit RESUME after the flash? if you remember James said it always halts at the first instruction when you flash debug or release.  Look for the Little green arrow.  

Did your MAC show the USB device?

**Comment by cad-quark** [comment]
I just hooked up my new M1 Mac Air and I see the Mac does not make any sound when you connect the LaunchXL USB cable.  I did find it listed under SYSTEM INFORMATION It lists a USB 2 as XDS110.  It lights up LED 0, 1 and 3.  This new Mac has only a 3.1 round USB jack so I am using a USB adapter.  I do not have CCS9.1 installed on the Mac (can not figure out how to load it), since I used PCs to flash my LaunchXL.

**Comment by andrea271068** [comment]
Hi, thanks, yesterday, I make huge disaster, I have many errors now, and now the blue led in the LaunchXL is off. I can't find the way to reset and cancel. I wan to reset the LounchXL, please you know how to do it? In one of your previous e-mail you told that you was force to reset many time for some reasons, and you told about a white button, ? I wan also delete the CCS and the software yo run the ELS, I wan to start again from 0. Please can you drive me on this? Many thanks

**Comment by andrea271068** [comment]
I think that LounchXL is burned, when I connect directly to PC usb no light are one, and the computer say that he was force to disabilitate the USB because an accessory absorb to much energy. Also when I plug in the James PCB the LounchXL is death!! 
Now I purchase again a new LounchXL a new James PCB and a new Led&Key, I will remove everything from Pc and start everything again from 0. Please when the new goods will arrive, can somebody drive me true from step 1, now I have to wait few days

**Comment by andrea271068** [comment]
I'm going crazy, now is running again, but still found 38 errors, I din't know what to do, completely lost now. How I can erase the project in CCS and start again? Please drive me true. ![image](https://user-images.githubusercontent.com/75749307/108618290-3fe86300-741d-11eb-803a-42533005a3ab.jpeg)![image](https://user-images.githubusercontent.com/75749307/108618296-4aa2f800-741d-11eb-98d1-a5acefdc9339.jpeg)

**Comment by cad-quark** [comment]
The white reset button is on the edge just up from the center jumpers.  If your main processor were a clock, it would be at 11:00.  Lay a rule along the top in your photo and its at 4cm.

Again, I am no programmer, but it looks like some punctuation has been removed from main.cpp.  Did you edit it?   If it were mine, I would re-unzip the ELS files.  I did so several times when I got the “target not found” or XDS110 failed messages before I found my 5v problem.

I am sorry I cannot give you better support.

**Comment by cad-quark** [comment]
Its not void (void)
It should be void main (void)

**Comment by cad-quark** [comment]
At the first red x

**Comment by cad-quark** [comment]
Line 92.  Where did that “=“ come from?  I do not see it in my main.cpp

**Comment by cad-quark** [comment]
For power, the USB from the Mac or a PC CANNOT power any Clough booster boards and the LED board must take over 100 mA.  In fact I removed the middle ( 5v ) isolation jumper when I loaded the program to the bare LaunchXL-F280049c board.  

Build ( if the Hammer symbol is dark) Debug ‘Release’ or els-f280049c,  whichever you have AND hit RESUME.  Disconnect USB AND the isolation jumpers for ground, and both voltages if present.  Then put on the Clough42 boards and plug in 5v power to the booster board.  The LED board should show Clough42, ELS 1.301, then 0 rpm, 005 feed.

If you know all your assembly is good, and your 5v supply is correct, then you can load the program like James does in part 10 video with the LED board so you know if its working.  

The last picture you sent shows the LaunchXL powered.  Is it now burnt out?

**Comment by cad-quark** [comment]
As Export manager, you might know if its possible to send a computer board/development kit through the international mail.  The USA has rules about technological exchange in ITAR and EAR.  If we could figure that out, I could get another card from TI and flash it with your parameters and ship it to Gerbolina MN.

**Comment by andrea271068** [comment]
Ciao, many thanks for your support. I never touch the program, not even for mistake, my problems yesterday come when I remove the middle jumper, I did when the power was on for all three jumper, after that many errors start to come out. For this reasons I would like to erase everything, even the CCS and the ELS 1.3..., everything!in this way if something is corrupted I will jump the problem.

**Comment by andrea271068** [comment]
Regarding the LounchXL, I bought a new one from US, TI web site, should arrive within two weeks, if I will find same problems, sure man, Would be great if you can send to me a preinstalled board. i will pay for everything also for the bother that I gave to you, I'm force to find any solution to pass true this, I spend a little fortune for my hobby and I'can't stand with the machine not running with the ELS

**Comment by andrea271068** [comment]
Today I did others staff, I was little I've had enough! I say to my self give little time to you on this issue, is drive me crazy, and this kind of panic is not good to see clear. Today I start to make a back plate for my lathe for a 4 jaws chuck and other little fixing on my mini mill machine. 
🙂

**Comment by cad-quark** [comment]
if the jumper was removed under power, it might have introduced some errors, I do not really know.  I would reload everything.  I did on my PCs a number of times.  I have been working on this for weeks, and gave up several times.    We will get there, somehow. 
 Can you tell me how to load my new Mac with CCS and ELS?

**Comment by cad-quark** [comment]
by the way, what in on your roof at the plant?  Solar panels?

**Comment by cad-quark** [comment]
just for fun, what is your encoder and stepper resolution and your thread pitch?

**Comment by andrea271068** [comment]
I found the white button, but how to do it? I push but notting happen. I open code composer studio, I open ELS project, and I push the white button in the TI board, but notting happen, may that is not the way to do it... 
many new thighs for me, I never deal with programs and good electronic level jobs. But I have good manually almost for everything and even if in my life I did something completely different, my school education was for mechanic engineering. 🙂

**Comment by cad-quark** [comment]
When I had the LED2 blink I pushed the white button and the LED stopped it irregular flashing.  Other than that, I do not know.  If the LEDs still come on on the LaunchXL board then it still may be good.  Remember isolation jumpers on when using USB power from the PC/Mac but only if all the other boards and cables are disconnected.   If you use the 5V power plug, then the isolation jumpers must be pulled off the LaunchXL board.  That is the way James loaded the program so he could immediately see the LED board show this program running after he flashed and hit RESUME.

**Comment by andrea271068** [comment]
On top my ruff? Solar panels. 
I just download the CCP from It for Mac, and for the ELS I sace in the mac book in a folder, just that! Was easy, may too easy 🙂

**Comment by andrea271068** [comment]
I have a nema 23 close loop, for the encoder same a James, I have a metric lead-screw 2 mm pitch

**Comment by andrea271068** [comment]
Manuale nema 23 close loop 
￼
HSS60驱动器-说明书.pdf

**Comment by andrea271068** [comment]
![image](https://user-images.githubusercontent.com/75749307/108640109-233d4100-7498-11eb-9a32-354dd3f23215.png)

**Comment by andrea271068** [comment]
![image](https://user-images.githubusercontent.com/75749307/108640125-33edb700-7498-11eb-901d-2783fc622ef3.png)

**Comment by andrea271068** [comment]
![image](https://user-images.githubusercontent.com/75749307/108640134-3ea84c00-7498-11eb-963d-5b612337f479.png)

**Comment by andrea271068** [comment]
![image](https://user-images.githubusercontent.com/75749307/108640143-4ec02b80-7498-11eb-892f-73b3ca2f361a.png)

**Comment by andrea271068** [comment]
![image](https://user-images.githubusercontent.com/75749307/108640152-5c75b100-7498-11eb-8e5c-b2044fab2adf.png)

**Comment by andrea271068** [comment]
![image](https://user-images.githubusercontent.com/75749307/108640166-68fa0980-7498-11eb-8f1c-6d58ab23d85d.png)

**Comment by andrea271068** [comment]
What parameter I should put in Configuration.H? 
And in the driver I leave the 400 micro-step as for default, but in Configuration.H is right for my case to place 1000 steps as James did for his?

**Comment by cad-quark** [comment]
I have tried several times to load CCS to the Mac, but it tells me I do not have permissions to install it.  This is probably one of those false "we do not know why it broke- so we will spit out something" messages.  In 'Users' it shows me to be an admin.  It un-zips and creates a directory, but the app does not run.

 I plan to use 1000 hybred stepper steps and 4000 like your encoder which is  A+,A-,B+,B- : so four pulses times 1000.  I am waiting on a 36v stepper supply.

I think somehow your Main.cpp got corrupted.  You may want to reload all ELS 1.301. 

I see its midnight in Milan.  I can send you text files if that line in main.cpp 'void main ( void )' does not show up.

**Comment by andrea271068** [comment]
Many thanks, yes try to send to me the text for the correct Void. Anyway I will try again to install ELS. 
Only one things, before the File in CCP when corrupted, I succeed to flash the firmware in the LaunchXL, now how I can delete that flash in the permanent memory if the LaunchXL? Or is not necessary because-if I will reload and debug again the new version will also remove the previous one?

**Comment by andrea271068** [comment]
Finally!! 🙂 I reload the ELS!! And flash to the TI LounchXLXL as for your instruction, now the CODE is running and flesh in the board!! No more errors. But still the PCB clught42 doesn't work!! Now I solve the Code problems, I'm quite sure the LounchXL is working, the CCS work, the only things is the PCB, I wan to try to desoldering all the connection elements, and solder again back, or is a soldering problem or may the PCB is defective. What you think? You have others idea or suggestion? Anyway I'm happy because the code is running or at least not errors are shown. 🙂

**Comment by andrea271068** [comment]
This is the result![image](https://user-images.githubusercontent.com/75749307/108674567-c9636800-74e5-11eb-9bc2-77967840404e.jpeg)

**Comment by andrea271068** [comment]
![image](https://user-images.githubusercontent.com/75749307/108674594-d6805700-74e5-11eb-9eb0-f83ffd918bc5.jpeg)

**Comment by cad-quark** [comment]
It looks like your isolation jumpers are still on the LaunchXL board.  Remove them if you also power from the 5v plug.

Did you hit RESUME after the flash, the green arrow in CCS?  Very important.

Finally, have you used Clough's 5 circuit tracing photos to check your soldering?

**Comment by cad-quark** [comment]
You must have power to the LED board since the one LED is on, but the other three wires should be checked.  Did you use the no-lead soldier?  I did, but I checked it very carefully.

I have been using a static wrist strap to handle the TI board as we are in winter and the air in the house is very dry.  You probably do not see dry conditions at all.  The only dry you have is the fine dry white wines from Mezzacorona

**Comment by cad-quark** [comment]
Two things I do not understand:   The resume button is green, like it has not been clicked, and the "Build" hammer is dark.  My Build symbol is light after the flash since the build is done for the files.  Your arrow is on the halt button.  Did you halt the program?

Also: You might want to spin the encoder to see if you get an RPM reading.

**Comment by cad-quark** [comment]
. . . After you make up the encoder cable.  

Did you get your backplate done?

**Comment by andrea271068** [comment]
I ‘m in office now 😊
But later I will back at home, and I will try.
Amway yes, I resume and after to disconnect I click the red square button.
The photo that I sent to you was after the flash, I debug again but not in the flash because I forgot to take the photo but is different and this is why you comment in that way.
Later at home I will write you again.

No yet for my back plate, I work Monday to Friday, some time from my hose in smart working, some time in office.
Were you from in US?
I leave 4 years in NY, and I have family there.
US is a second home for me but is more than 3 years that I didn’t come in US. Hope soon to organize a nice visiting there.
I love USA and in my business life I visit all the world.
US, Japan are my favorite, but also some other Asia country are very nice.

Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by andrea271068** [comment]
Regarding the vine 😂, is true, here is very humid lot of fog during winter time. I can wait to have an aperitive🙂

**Comment by cad-quark** [comment]
I assume the S3 and S4 switches are set correctly.  Also S2 (boot mode ) should be at default (both off) the way it came.  Be careful here as TI put S2 on upside down ( TI users guide item 2.1.6) so off is logic 1.  That is the boot from flash setting.

**Comment by andrea271068** [comment]
You know, to take out any doubts, later ai wan yo install CCS in a window Pc and try from there, my nephew will come with his computer, at home we have Apple mac book, I can't use my work PC, personal installation are prohibited and not possible. 
For the soldering I use no lead, I scared my that I use to much fluxing paste. I truth I wan to try check each soldering. Now my nephew cime and I wan to try to install from his Pc, I will let you know, 🍷

**Comment by cad-quark** [comment]
Here in Iowa its winter and after several days of below freezing it is starting to warm up above 0c.  Our Meter thick snowbanks should shrink this week.  I am a retired Avionics Systems test engineer at Collins, now United Technologies in Cedar Rapids.  The farthest south I got in 1969 on the college graduation tour was Zurich.  Then we rode my BSA to Chartres to camp for a few days while we took the train to Paris.  Bad timing.  That was August and everything was closed.

**Comment by andrea271068** [comment]
This is the my LounchXL, check please the settings ![Uploading 108752278...]![image](https://user-images.githubusercontent.com/75749307/108752378-ac5d8200-7543-11eb-890b-dd4d01882b5d.jpeg)![image](https://user-images.githubusercontent.com/75749307/108752417-b8494400-7543-11eb-80e2-edf7718b247e.jpeg)

**Comment by andrea271068** [comment]
You say: Also S2 (boot mode ) should be at default (both off). Where is the off? This is like is set my board, is correct? Is strange I din't remove the protection tape over the switch,  but this switch is without tape, may this board was already used? This is why I think maybe is not as for default setting, please check the photo that I sent to you, thanks

**Comment by andrea271068** [comment]
Wow you was making a great job, avionics, very nice and fascinating for me, for almost 30 year I take hundreds and hundreds  planes ✈️ to flight allover the world, one of my hobby is model building but this have notting to fo with avionics, but can give you an idea how much is fashionable for me your job, do with airplanes industry is great. Is more then one year bow, thanks... to covid that I didn't travel, I can't wait to travel again, and be in the airplane again.

**Comment by cad-quark** [comment]
S2,S3,S4 looks like my board.  I got my board from Digikey and there was no tape on any switch.  I might try buying from Mouser the next time, and I may as well get the Clough42 boards again.  If everything goes well, I may spring for a new Lathe and sell my multifunction Harbor Freight mill/lathe and I will want an ELS for that.

What I tested were mostly Bizjets for HIRF and Lightening but not from Boeing or Airbus.

Did you get a chance to check the connections from the booster board to the LED&key board?

**Comment by andrea271068** [comment]
I test the continuity in the wire and look fine

**Comment by cad-quark** [comment]
Have you tried flashing like Clough did in part 10 with the isolation jumpers off, using your 5v source?  You may have to make a small change so CCS sees a reason to rebuild the program.  Maybe an additional comment like : // add another do nothing line.
 
The compiler probably throws the comment lines away, but it might see the program file has changed so it will build, link compile and flash.  I may try that here.

**Comment by andrea271068** [comment]
Last Saturday I did the same, I was so frustrated sick and tired, that I please too the PCB in the wrong position, I think all the errors was also for that reasons. Now I'm downloading CCS in a window Pc, I wan to flash the cose using a window machine. Let' see. 
Before you say that I should try to add in CCS some command? I din't know how to do it. In case needed you can drive me? Or send a sample page with the command and pisitions?

**Comment by cad-quark** [comment]
Looks like LaunchXL blue LEDs show programming.  LED3 is on with power, LED2 is on during and after Flash GEL

**Comment by cad-quark** [comment]
Which is “General Extension Language”.   It looks like you do not need any changes to rebuild and flash.  Just click the debug bug.  You must have changed the Configuratioh.h when you choose 200 mm for your ‘pitch’ selection.

**Comment by cad-quark** [comment]
Good luck with the PC.  Be aware some Windows 10 pcs can be ‘S Mode’ and do not accept any software but Microsoft.  Another reason to despise Microsoft.

**Comment by andrea271068** [comment]
I dis, but still PCB doesn't work, this is the all sequence. Check the photo step by step. I think the code is flashed in the LaunchXL, the two blue light are on, but the PCB not start. The good things not errors message show-up.now really I start to think or the connections are not working or the PCB is defective!! ![image](https://user-images.githubusercontent.com/75749307/108779043-9ad9a180-7566-11eb-972c-0281802e541b.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779071-a4630980-7566-11eb-8316-59aacd739eec.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779094-acbb4480-7566-11eb-9578-3a9568f0e4fd.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779104-b17ff880-7566-11eb-9a87-d01c09dc3191.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779117-b47ae900-7566-11eb-9052-fc5136b313d1.jpeg)![image](https://user-![Uploading 108779162...]images.githubusercontent.com/75749307/108779140-bb096080-7566-11eb-9147-f0748078b057.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779220-d5433e80-7566-11eb-9069-63bf7f407463.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779267-e724e180-7566-11eb-95b6-5eeb33d938fe.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779286-ec822c00-7566-11eb-8e14-9398ca3eb812.jpeg)

**Comment by andrea271068** [comment]
Wow!!  it is very strange after few minutes this error show up??? And now In the LounchXL??? I was happy that now errors show up, but after few minutes on appear, very strange, lost again!!!!! 

![image](https://user-images.githubusercontent.com/75749307/108779469-3539e500-7567-11eb-8bd5-163e642a9dc6.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779519-4a167880-7567-11eb-9b53-a30e85d42fad.jpeg)![image](https://user-images.githubusercontent.com/75749307/108779576-5c90b200-7567-11eb-90a9-43cbe14f8ea5.jpeg)ave only one blue light in th

**Comment by andrea271068** [comment]
Now I try to do like you told me, without PCB and without the middle jumper, and the code run in the flash without problems, but when I disconnect and I place the PCB removing the three jumpers, the same and the second blue light is off same as PCB. I'm aure that the PCB is defective or bad soldering connection, in the lounchXL how many red light you have two? Or three?

**Comment by andrea271068** [comment]
Yes I push resume, I will desolderimg everything and solder again, ot is strange because I check everything with the tester as Mr. James say,  and was okay

Scarica Outlook per Android<https://aka.ms/ghei36>

________________________________

**Comment by JonWoellhaf** [comment]
You can usually leave S Mode. See https://support.microsoft.com/en-us/windows/windows-10-in-s-mode-faq-851057d6-1ee9-b9e5-c30b-93baebeebc85.

**Comment by andrea271068** [comment]
Hi, but when the code is permanent flash inside the Lunch XL the LED2 should be permanent on, even if I power off the board?.
When I will switch on again LED3 and LED2 have to permanent On or just LED3 will be on?

Because when I power off after the permanent flashing when I power on again only LED3 is on.

I wan to make sure that the code is flash inside the memory in a permanent way, PCB doesn’t work and I don’t know if the code is inside the LaunchXL or not.

Many thanks


Andrea Pisani
Export Manager

Composad srl – Italy
Mobile Phone: 0039 335 70 98 102
E-mail: a.pisani@composad.com<mailto:a.pisani@composad.com>

**Comment by andrea271068** [comment]
This is may back plate ![image](https://user-images.githubusercontent.com/75749307/108839650-a31cf580-75d5-11eb-9204-186af2a5c264.jpeg)

**Comment by cad-quark** [comment]
My LED3 is on with power, LED2 is on with and after Flash.  If power is removed, LED2 will stay off.  

Jonwoellhaf; we are stumped.  Andrea has flashed the .out file without any connections but LaunchXL and the Mac, ( to eliminate soldier and cable problems ) but the program will not show up.  He also tried a HP notebook in case Mac does not work, he also reloaded the version 1.301 to workspace.  

Andrea: did you load CCS 9.1 to the HP?  I do not think CCS 10 works.

**Comment by andrea271068** [comment]
Ccs 9.3

**Comment by cad-quark** [comment]
I see it is Workspace V9, so you probably loaded CCS9.

I just bought the Clough42 boards again.  If you do not resolve the problem I can assemble, test and ship them in a few days.  Will they need to go through customs or any special handling?

**Comment by cad-quark** [comment]
Nice mill:  VFD drive, ER collets, DRO, mist.  I see a compressor and Tig torch as well.  When the 4-jaw is mounted you will join Turnwright and ABOM79, and hopefully turn threads without change gears, which they can not do.

**Comment by andrea271068** [comment]
Todayva receive a new LounchXl, I order too a new Clught PCB, and now I'm waiting, I thought the same, let's try with this second trial issue, if I can't solve even with that, I will ask you the kindness to send a preinstalled kit to me, I din't thinks that we will find problems at custom, sure we have to declare something, let's see

**Comment by cad-quark** [comment]
Right.  I hope the first Launch XL is good and that the error is within the external stuff.  I see my Clough42 has shipped.  Its melting here in Iowa City, and the ice is sliding slowly off the metal roof.  Life is good.  Now if I can get a Covid19 shot. . .  Are you able to re-borrow that HP notebook? However I do not think the Mac is the problem.  I just wish I could get my Mac to load CCS.

**Comment by andrea271068** [comment]
Ho, please check this, I try to teat continuity within the Led&key I test as for thr James continuity chek the dio clk stb, but not signal, is correct? No signal? Check the photo may is better than any words !

[Uploading 108883725...]![image](https://user-images.githubusercontent.com/75749307/108883921-4b4ab280-7606-11eb-9bdf-9b8f17d62d10.jpeg)

**Comment by andrea271068** [comment]
![image](https://user-images.githubusercontent.com/75749307/108884010-63223680-7606-11eb-9a52-45dbb72c09a4.jpeg)

**Comment by cad-quark** [comment]
Ok. I will check what I have there.  Ebay has a tiny oscilloscope the size of the Clough42 booster board.  You may need to order one of these, unless we can see a frequency from the serial lines. Its 1700 where you are, so I will try to get some readings before you turn out the lights.

**Comment by cad-quark** [comment]
I get a pulse from all three data lines, but the volt meter is too slow to see any DC but 004v.  You need an O-scope.

**Comment by cad-quark** [comment]
![F3AD0A3A-A87E-4295-B87B-19BA4770FEC2](https://user-images.githubusercontent.com/75825422/108898253-9bad1a80-75dc-11eb-9f90-bdfd80b60120.jpeg)

**Comment by cad-quark** [comment]
Digital scope $25 from Ebay.

**Comment by andrea271068** [comment]
?

**Comment by andrea271068** [comment]
Nice, cool

**Comment by cad-quark** [comment]
![05F18787-C2B8-45EF-A284-885C4C34695B](https://user-images.githubusercontent.com/75825422/108901213-365b2880-75e0-11eb-839c-ec12ddcac4ec.jpeg)
Stb
![50B31CF7-916B-461C-AB5E-C89A046DE2A0](https://user-images.githubusercontent.com/75825422/108901373-69052100-75e0-11eb-8012-c45639918bd2.jpeg)
Dio

**Comment by andrea271068** [comment]
I have a question, if for this moment I wan to use the servo to run just the leadscrew for power feeding what I should do? I need just to buy a simple pulse generator with forward and reverse? Or I need to buy also other stuff, Arduino etc. Can somebody list to me what I should buy for my servo? I wan a simple momentary solution, at least I can power feeding and use my lathe. I don't wan to disassemble the ELS set up and put back gears.
For my nema 23 hybrid servo setup. 

IS NOT MY SURRENDER!!!

**Comment by andrea271068** [comment]
What is your name?

**Comment by cad-quark** [comment]
Find an old PC with a parallel output, download Mach3Mill to it then with a cheap interface board hook up the one servo.  Mach3Mill has a jog function or you can give it simple G code commands to get your feed.  I plan to have ELS and Mach3Mill in place on the same mill/lathe even if I have to remove drive belts to do operations.

**Comment by cad-quark** [comment]
Chuck

**Comment by cad-quark** [comment]
In G code the lathe axis is Z and the cross feed is X, where milling is X-Y plane and Z down.  The program does not need to know.  There is a Mach3Turn that can do threads as well

https://www.machsupport.com/wp-content/uploads/2013/02/Mach3_Threading.pdf

**Comment by JonWoellhaf** [comment]
I just ordered one of those DS0138 scopes. I got ripped off, though. Mine was $31.61, total. 

I think anyone who attempts James’ ELS, and doesn’t already have a scope, should buy one of these.

**Comment by cad-quark** [comment]
I could widen out the pulse but I have no clue what it does anyway.  I just can see some kind of data is present on all 3 lines while it works.  My 20M analog scope cannot capture the pulses.  
My theory is that one of Andea's ROHS 'soldier' joints is high resistance.  I used the crapy no-lead but I checked pin to pin.  I will check the boards I send to Italy.  I'm betting his LaunchXL works but cannot communicate to display.

Jonwoellhaf are you in the same time zone as Andrea?

**Comment by andrea271068** [comment]
Thanks guys

**Comment by cad-quark** [comment]
I was able to install CCS9.3, and it a lot different from CCS9.1.  It has fewer code options, not that I know anything whatsoever about options.  I may wait for my new LaunchXL-Fxx before I try this CCS flash.  It does put a shortcut on the desktop, and also a  eclipse icon in the tray.  CCS was bought by TI when it was called CC.  They were working on the Eclipse and combined the two.  That explains some of prolific name structure.   Now for the Mac loosey goosey file system for a workspace.

**Comment by andrea271068** [comment]
?

**Comment by cad-quark** [comment]
got a big blank CCS workspace and put 1.301 in it but no change.

**Comment by andrea271068** [comment]
You are saying that I should try with CCS9.1? 
I can download and check if this can be the problem, yesterday I desoldering PCB and Led&key, today I wan to put together again and try

**Comment by cad-quark** [comment]
all I know for sure is they are different.  I get the impression we are using a CCS shot gun to kill house flys.  CCS is vastly more powerful than our use as a eeprom writer.

**Comment by cad-quark** [comment]
Oh yes, and the quark thing.  Its teletype shorthand for question mark. My first computer was an 8-bit Z-80 running CP/M

**Comment by cad-quark** [comment]
Re: Mac load.  The CCS9.3 does not see any files in the workspace_v9 where I put ELS 1.301.  I used the Zip file, should it be the .tar file since Mac OS is unix based?

**Comment by andrea271068** [comment]
My work space is in documents. Just there and look working LounchXl will unpack from that location, sure you have to drive him there

**Comment by andrea271068** [comment]
Maybe I found a connection problem on the Led&kay in Dio, Stb and the other, no continuity from the input pin and their destination, a friend of me tell me to follow the micro wire and scratch the board in the correspondence, and when I test continuity, the tester sound, I think there is a problem in the board, I can wait to modify the board tonight to trace a new wiring connection and test if will run, I really hope, thanks God my we found the problem, let's see...
Yesterday I say that when i test so sound was coming out, no continuity, I also send the photo. But thanks to that scratch now I know that should be close the continuity.

**Comment by cad-quark** [comment]
Lets hope that is true.  I will not uncork the Bottega Vinaia yet.

**Comment by andrea271068** [comment]
I will be happy to offer you 6 pack.. 🙂

**Comment by JonWoellhaf** [comment]
I’m in Utah, USA. GMT –7 -- at least until we begin the stupid daylight saving time.

**Comment by andrea271068** [comment]
Thanks Jon :)

**Comment by andrea271068** [comment]
WOW!!!!! We get it!!! The problem was in the Led&Key board!!!!! 

Now I have to place inside the enclosure and connect with all the rest and make a run test in the machine!!! I'm soooooo happy 

In place there the photo of what I did for everybody   😅😅😅 thanks to everybody!!!   

The code was flashed, but before to say done let's wait the installation in the machine enclosure, never know... 
 ![image](https://user-images.githubusercontent.com/75749307/109138566-cc1ac300-775a-11eb-81b1-29df230daefa.jpeg)

![image](https://user-images.githubusercontent.com/75749307/109138600-d5a42b00-775a-11eb-9bfc-73ae0e8b48df.jpeg)

**Comment by andrea271068** [comment]
It's running but is not stable, I have interferences may I have some noise, or some not stable connections and with the vibration move, I will check everything, but I'm very happy now!!! 

Thank you Guys, thank you cad-quark, you didn't leave me alone, in the terrible mess timing, and with you I was able to analyze the things in a more stable way, I didn't was sure about me because I din't have electronic and programming competences! Thanks again, let's follow next steps, I will miss our communications.

**Comment by andrea271068** [comment]
Thank you Guys, thank you cad-quark, you didn't leave me alone, in the terrible mess timing, and with you I was able to analyze the things in a more cold and sure way, I didn't was sure about me because I din't have electronic and programming competences! Thanks again, let's follow next steps, I will miss our communications.

**Comment by cad-quark** [comment]
Cool.  Place the blame on RoHS solder.  Real lead soldier is so much easier to use.  I see James suggested more careful circuit tracing.  Another thing I would like to see is a debug trace or watch setup in CCS to monitor some variable so the LaunchXL board could be checked without any external hook-up other than USB.

**Comment by andrea271068** [comment]
Anyway didn't was this time for solder, even if now following you suggestion I solder  with sn 60%. 
Check the photo, what i did!! 
 
The LED&Key was defective and drive us crazy!! ![image](https://user-images.githubusercontent.com/75749307/109167037-a0f59b00-777d-11eb-8cdd-7699ed72a3c8.jpeg)

**Comment by andrea271068** [comment]
I have little noice problems, maybe is related to the fix board, but at lear now is working. Now I wait the new led&key, mid time I'm re wiring everything. Dud you have some suggestion for the noice? Did you see my enclosure photos. 
Thanks 😊

**Comment by cad-quark** [comment]
Congratulations.  You really did amazing work finding the problem.  It might still be RoHS soldering problem, but not your work, but whoever did the work on the production board.  

I think for the next round, I am going back to 60-40 leaded solder.  As such it cannot be sold in Europe.  If you still need the parts for reliability, I will give them to you.

**Comment by cad-quark** [comment]
Your enclosure is neat and it looks like it can be chip free behind a sealed cover.  Lathe chips, especially cast iron and brass seem to spread everywhere and might need silicon gaskets on the pc board enclosures.

**Comment by andrea271068** [comment]
Thanks, I think is protected. I hope to find the noice, Now I don't wan to go crazy because the Led&key is temporary now, is like a guys after an surgery 🏨, can be that the noice problem. I will know only after I will assemble the new led&key. How about you mac book set-up?

**Comment by cad-quark** [comment]
I kind of gave up.  CCS9.3 on the Mac refused to find the Clough42 v1.301 files and CCS9.1 will not load.  I erased everything.  I may spend some time on Youtube to learn about Mac big sur before I try it again.  I also may try Windows10 and CCS9.3 Since my Windows8.1 machine will flash 1.301.

I am waiting on James's boards to solder up if you still need them.  Will you use lead solder for your new boards?

**Comment by andrea271068** [comment]
Hi, I don't beed the Pcb board, I bought too, and should arrive in few days. But fid you buy for me? How I can refund? And now what you will di with that extra board? 
I use 60/40 lead.

**Comment by cad-quark** [comment]
No problem.   My long term goal is to buy a new bench lathe which will need an ELS.  I will also have a Mac with CCS.  I have been working with these boards since early December so time is no issue.

**Comment by cad-quark** [comment]
You may want to send the LED board to James as there seems to be a production error he should know about.

**Comment by andrea271068** [comment]
I think he know, I send the photo. At the beginning he answers to my post.

**Comment by andrea271068** [comment]
Dine!!! Also the noice look 90% over, some just one blink at the start 
Check the final result 

![image](https://user-images.githubusercontent.com/75749307/109341001-0921ab00-786a-11eb-9361-d659f4bb2cd7.jpeg)

![image](https://user-images.githubusercontent.com/75749307/109341028-18085d80-786a-11eb-9fb3-c00c5394c162.jpeg)

![image](https://user-images.githubusercontent.com/75749307/109341050-22c2f280-786a-11eb-87ee-9864354b4579.jpeg)

![image](https://user-images.githubusercontent.com/75749307/109341067-28b8d380-786a-11eb-8d82-3a8f6f27141a.jpeg)

![image](https://user-images.githubusercontent.com/75749307/109341080-340bff00-786a-11eb-9434-98df0fed12f1.jpeg)

![image](https://user-images.githubusercontent.com/75749307/109341132-438b4800-786a-11eb-9862-ec6156550222.jpeg)

**Comment by cad-quark** [comment]
Looks great!
  What was your final stepper setting in configuration.h?  It looks like 5:1.  Would that be 5000 steps?

**Comment by andrea271068** [comment]
I think 1:3, but when I watch Metric lead screw I see that James remove the 1 and replace with three, and I did, but us very slow. 
I have to do some test for that, I need to test on trading screw

**Comment by andrea271068** [comment]
You know as my next job may I would like to change the engine and drive with VFD, what you suggest three fase or two, my ELS setup is for two face. 

I'm waiting g a new great deep project from James, I would like to challenge my self again in a new serous adventure. 😁

**Comment by cad-quark** [comment]
I have a TB Woods as a VFD drive now and I have used most of the brand names below.  Most VFD drives on Ebay will take single phase 220VAC and output 3 phase variable frequency.

Ebay has Fiji, GE, Toshiba, Danfoss, Reliant, and all the no-brand Made in China boxes. Some brands have manuals on line.  You will need to buy a three phase motor and look at the manual of the drive to be sure you can run on single phase.  Usually you use the first two terminals of x,y,z and hook the motor to u,v,w.   Then you can set the parameters to run as high as 400hz.  Most motors can run at 120 hz easily, or if your standard is 50hz normally then push it to 100hz.  Torque falls off below 25 Hz, but that may still work for light loads.  There will be a terminal board to wire up stop, start, jog and for the potentiometer for speed change. You can set start and stop times,and hundreds of other parameters.  
Your home system, I believe is already 220v single phase @ 50 hz. I have not bought any of the no-name vfd units so I do not know how they compare.
Chuck.

**Comment by cad-quark** [comment]
Your ELS is two phase DC, maybe up to 50v.  The main motor is usually Single phase AC 220v.  It looks like you already have variable speed of some kind.

**Comment by andrea271068** [comment]
In y
Italy we have domestic Ac 220v 50Hz

**Comment by andrea271068** [comment]
I would like to have note HP, and the actual motor, is not brilliant, I have the sensation that the electronic is very cheep, I don't like, sound, and is not stable at all, the motor have some kind of peak, slight up and down. I see James video his lathe run with power qnd smooth. I don't like also my pulley to small, the belts are less the 6mm triangular.

**Comment by andrea271068** [comment]
In ebay there are those kind if set ![image](https://user-images.githubusercontent.com/75749307/109363702-785cc680-788d-11eb-8728-2b51f59db0cd.jpeg)

**Comment by andrea271068** [comment]
What difference there is from that kind of motor and a spindle motor? Like this below in the photo? ![image](https://user-images.githubusercontent.com/75749307/109363978-1baddb80-788e-11eb-93f7-134f20ca800e.jpeg)

**Comment by andrea271068** [comment]
What is better for a lathe? the spindle look more modern technology, and is more compact! And I like the compact idea! Will be more easy to find the spot.  I know spindle are use for the mills but can be use also for a lathe? And in case would be better then a regular motor three phase drive by Vfd?

**Comment by cad-quark** [comment]
I have a 1 hp 3 phase motor (12mm belt)on this lathe, but I had a 1.5 HP on my 12 inch Clausing.  The VFDs allow small rpm changes for best cut.

**Comment by andrea271068** [comment]
I crazy idea As next project, I would like to build WW2 radial engine. May I'm crazy, but I wan to do!!! I wan to try this adventure. 

 ![image](https://user-images.githubusercontent.com/75749307/109364902-1fdaf880-7890-11eb-9e13-9549aad5415c.jpeg)

**Comment by cad-quark** [comment]
The spindle is for a mill where you might use several thousand rpm on a small cutter.  Also the spindle uses ER collets limited to smaller diameters and will not allow long bars to be chucked through the headstock.

**Comment by cad-quark** [comment]
That should a real challenge.

I am now working on an EasyLift, made in Brescello.  A tree trimming service here uses it as an arial bucket.  Its all relay logic and has nearly 200 total switches, relays, lights and micro switches.  Its used in all weathers, like today at 3c and in the melting snow and mud.  It is 7 years old and starting to have electrical problems.  The wiring diagram is 30 pages long.

**Comment by andrea271068** [comment]
Wow!! Brescello ia two KM from were I live. Is next to me. 
This is funny.

**Comment by cad-quark** [comment]
Address is
Via Leonardo da Vinci, 10-12-14-16, 42041 Brescello RE, Italy

**Comment by cad-quark** [comment]
Phone
+39 0522 967620

**Comment by cad-quark** [comment]
I do need an updated wiring diagram for an R210 / 70-36aj

**Comment by andrea271068** [comment]
I will call the on Monday, and I will let you know, send to me your e-mail address and I tel them to replay to you. 
I need your contacts references

**Comment by cad-quark** [comment]
Email:cadewey@sprynet.com.  The sales director is the very nice Manuela Vender who insisted I talk to Up Equipment in Montreal, but the newest wiring diagram Up Eq. sent me did not have all the relays we see on the 2014 R210 lift.

**Comment by andrea271068** [comment]
👍 let ne see, I would be more than happy to helping you in your need. 🙂

**Comment by cad-quark** [comment]
![ED30FE9E-8DC3-4C62-AF1C-C8B4A37189C5](https://user-images.githubusercontent.com/75825422/109404528-a5ac8180-792c-11eb-83a9-84bc3962d553.jpeg)
My wiring diagram does not contain k32t or k15b, so I suspect it had been upgraded to a newer style.

**Comment by cad-quark** [comment]
I received my Clough42 parts.  Did you get your latest order?  I can easily soldier the Led&key board, test it, and ship it to you.

**Comment by andrea271068** [comment]
I call and talk with the service guys of easy lift, he ask about your e-mail address and I gave ti him. I explained your need, and he say that will write to you, but he say that here in US there is their trader

**Comment by cad-quark** [comment]
Thanks.  I have contacted the Canadian shop, but there is no US dealer.  I will get the Email, and reply all the details.

**Comment by andrea271068** [comment]
The service name guy is Gianluca Fes

**Comment by cad-quark** [comment]
Thanks.  The lift wiring is probably the hardest trouble shooting I have ever done.  Not having an accurate drawing is only part of the problem.-----Original Message-----
